.github/workflows/frameworks.yml

```diff
@@ -113,7 +113,7 @@ jobs:
               - *runner
               - bentoml/lightgbm.py
               - bentoml/_internal/frameworks/lightgbm.py
-              - tests/integration/frameworks/test_lightgbm_impl.py
+              - tests/integration/frameworks/test_frameworks.py
```

Comment:
we should add this under `related` keys so that all frameworks can have this by default.

Reply:
Probably after we convert all the framework tests over, though.